src/mono/wasm/runtime/marshal-to-cs.ts

```diff
@@ -49,6 +49,66 @@ export function initialize_marshalers_to_cs(): void {
     }
 }
 
+export function bind_arg_marshal_to_cs(sig: JSMarshalerType, marshaler_type: MarshalerType, index: number): Function {
+
+    if (marshaler_type === MarshalerType.None || marshaler_type === MarshalerType.Void) {
+        return () => undefined;
```

Comment:
```suggestion
        return (args: JSMarshalerArguments, value: any) => undefined;
```
It does actually matter what the signature of the JS function receiving a call is for a few specific things, so it's worth being correct here

Reply:
I got rid of NOOP functions